pyrk/materials/mod.py

```diff
@@ -0,0 +1,87 @@
+from ur import units
+from material import Material
+from density_model import DensityModel
+
+
+class Mod(Material):
```

Comment:
Again, we already have a "graphite.py" material to represent graphite. Moderators can be many different materials. Redundant classes with generic compoent-level names are not appropriate for additions to the material library. The material library should contain nuclear materials like "SS316" "Sodium" or "LiquidSodium" or "flibe" or "graphite" since reactors use very different materials for different components. We can talk about this later to make this concept more clear if necessary.